### PR TITLE
adapt for workflow testing

### PR DESCRIPTION
**Pull request by bernt-matthias**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -17,22 +17,33 @@ inputs:
...
     description: 'List of repositories.'
     default: ''
-  lint-tools:
-    descriptions: 'Lint planemo tools?'
+  tool-list:
+    description: 'List of tools'
+    default: ''
+  # Mode selectors
+  lint-mode:
+    descriptions: 'Run lint mode?'
```
Comment (mvdbeek) [inline action.yml:28]:
```suggestion
    description: 'Run lint mode?'
```

File: action.yml [hunk 2]
```diff
@@ -17,22 +17,33 @@ inputs:
...
+  test-mode:
+    description: 'Run test mode?'
+    default: false
+  combine-mode:
+    descripttion: 'Combine multiple test reports into a single output'
     default: false
-  test-tools:
-    description: 'Run planemo tests?'
+  check-mode:
+    descripttion: 'Check test report for errors'
```
Comment (mvdbeek) [inline action.yml:37]:
```suggestion
    description: 'Check test report for errors'
```

**Review by mvdbeek — COMMENTED** [review]
That looks great!

File: action.yml [hunk 3]
```diff
@@ -17,22 +17,33 @@ inputs:
...
+    default: ''
+  # Mode selectors
+  lint-mode:
+    descriptions: 'Run lint mode?'
+    default: false
+  test-mode:
+    description: 'Run test mode?'
+    default: false
+  combine-mode:
+    descripttion: 'Combine multiple test reports into a single output'
```
Comment (mvdbeek) [inline action.yml:34]:
```suggestion
    description: 'Combine multiple test reports into a single output'
```

**Comment by bernt-matthias** [comment]
One thing that I thought about was to have a single `mode` option instread of all those `...-mode` options...?

**Comment by mvdbeek** [comment]
I think both is fine, that's up to you.

**Comment by mvdbeek** [comment]
Amazing, I love it!

**Pull request merged**